start_jupyter.sh

```diff
@@ -1,7 +1,29 @@
-if [ $UPDATE_ARK -ne 0 ]
+#!/usr/bin/env bash
```

Comment:
@srivarra should we keep the old version or leave it updated to match `toffy`?

Reply:
Yeah lets update it to match `toffy`'s.